dms/views/directory.xml

```diff
@@ -451,7 +451,7 @@
                     </group>
                     <group name="data">
                         <group>
-                            <field name="size" />
+                            <field name="size" widget="integer" />
```

Comment:
Isn't here a widget `dms_size`? Is interesting to be rescued? cc @etobella

Reply:
Any answer?

Reply:
Finally, do we merge this with `widget="integer"` or try to rescue `widget="dms_size"`?

Reply:
Let's go for now with the integer, although it would be interesting to rescue it.